components/edit_channel_purpose_modal/edit_channel_purpose_modal.jsx

```diff
@@ -46,30 +48,13 @@ class EditChannelPurposeModal extends React.PureComponent {
 
         this.state = {
             purpose: props.channel.purpose || '',
-            serverError: '',
+            serverError: null,
             show: true,
             submitted: false,
             requestStarted: false,
         };
     }
 
-    setError = (err) => {
-        if (err.id === 'api.context.invalid_param.app_error') {
```

Comment:
I didn't go back far enough to find the origin of this part of the code, but this wasn't even the right error ID that would be returned by the server if the user's input was too long, so I just decided to get rid of it entirely.

I also changed the type of `this.state.serverError` to be an object like most other places in the web app.